src/transformers/models/longt5/modeling_longt5.py

```diff
@@ -1391,11 +1385,11 @@ def __init__(self, config, embed_tokens=None):
         self.final_layer_norm = LongT5LayerNorm(config.d_model, eps=config.layer_norm_epsilon)
         self.dropout = nn.Dropout(config.dropout_rate)
 
+        self.gradient_checkpointing = False
+
         # Initialize weights and apply final processing
         self.post_init()
```

Comment:
here moving it before post init because `post_init()` calls `gradient_checkpointing_enable()` if in the config you have an attribute `gradient_checkpointing`